libs/templates/src/lib/trade/table/trading-spec-table.tsx

```diff
@@ -100,16 +98,18 @@ const TradingSpecTable = () => {
           <LabelPairedEllipsisVerticalBoldIcon
             fill="black"
             iconSize="md"
-            className="block md:hidden lg:hidden"
+            className="min-sm:block max-lg:block md:hidden lg:hidden"
```

Comment:
`min-sm:block` is not valid in tailwind.
```suggestion
            className="block md:hidden"
```